src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Reissuance/ReissuanceConsent/HealthCertificateReissuanceConsentViewController.swift

```diff
@@ -38,6 +37,16 @@ class HealthCertificateReissuanceConsentViewController: DynamicTableViewControll
 		self.dismiss = dismiss
 		
 		super.init(nibName: nil, bundle: nil)
+		
+		viewModel.$reissuanceCertificates
+			.receive(on: DispatchQueue.main.ocombine)
+			.sink { [weak self] _ in
+				guard let self = self else { return }
+				
+				self.tableView.reloadData()
```

Comment:
```suggestion
				self.dynamicTableViewModel = self.viewModel.dynamicTableViewModel
				self.tableView.reloadData()
```

You need to update the model first, otherwise it will just reload with the old model, not actually updating anything